qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -26,7 +26,19 @@ fi
 # Check that the kube of the cluster is reasonable
 bash ${CAP_DIRECTORY}/kube-ready-state-check.sh kube
 
-PROVISIONER=$(kubectl get storageclasses persistent -o "jsonpath={.provisioner}")
+if kubectl get sc | grep "persistent" > /dev/null ; then
+    STORAGECLASS="persistent"
+    PROVISIONER=$(kubectl get storageclasses ${STORAGECLASS} -o "jsonpath={.provisioner}")
+    cap_platform=$(kubectl get configmap -n kube-system cap-values -o json | jq -r .data.platform)
```

Comment:
Set this outside of the if statement:
```
cap_platform=${cap_platform:-$(kubectl get configmap -n kube-system cap-values -o json | jq -r .data.platform)}
```

Reply:
no, because there is no configmap named `cap-values ` in EKS clusters